# error: implicit declaration of function 'ldap_search_s' is invalid in C99 [-Werror,-Wimplicit-function-declaration]

--- src/browse-dnd.c.orig	2007-05-15 00:26:50.000000000 +0800
+++ src/browse-dnd.c	2022-10-24 22:13:08.000000000 +0800
@@ -33,6 +33,8 @@
 #include <glib/gi18n.h>
 #include <gtk/gtk.h>
 
+#include <ldap.h>
+
 #include "browse-dnd.h"
 #include "debug.h"
 #include "errorchain.h"

--- src/gq-browser-node-server.c.orig	2007-05-15 00:26:50.000000000 +0800
+++ src/gq-browser-node-server.c	2022-10-24 22:13:35.000000000 +0800
@@ -35,6 +35,8 @@
 # include <config.h>
 #endif /* HAVE_CONFIG_H */
 
+#include <ldap.h>
+
 #include "gq-browser-node-dn.h"
 #include "gq-tab-browse.h"
 #include "prefs.h"			/* create_edit_server_window */

--- src/gq-tab-browse.c.orig	2008-01-09 00:36:10.000000000 +0800
+++ src/gq-tab-browse.c	2022-10-24 22:13:49.000000000 +0800
@@ -36,6 +36,8 @@
 # include <config.h>
 #endif /* HAVE_CONFIG_H */
 
+#include <ldap.h>
+
 #include "common.h"
 #include "configfile.h"
 
--- src/input.c.orig	2007-05-15 00:26:51.000000000 +0800
+++ src/input.c	2022-10-24 22:14:12.000000000 +0800
@@ -38,6 +38,8 @@
 # include <config.h>
 #endif /* HAVE_CONFIG_H */
 
+#include <ldap.h>
+
 #include "mainwin.h"
 #include "common.h"
 #include "configfile.h"

--- src/browse-export.c.orig	2007-05-15 00:26:50.000000000 +0800
+++ src/browse-export.c	2022-10-24 22:07:02.000000000 +0800
@@ -37,6 +37,8 @@
 # include  <config.h>
 #endif /* HAVE_CONFIG_H */
 
+#include <ldap.h>
+
 #include "common.h"
 #include "gq-browser-node-dn.h"
 #include "gq-browser-node-reference.h"
